.github/workflows/deploy-mfe-s3.yml

```diff
@@ -0,0 +1,106 @@
+name: MFE S3 Bucket Deployment 🚀
+
+
+on:
+  push:
+    branches:
+      - open-release/palm.nelp
+      - open-rc/palm.nelp
+
+  pull_request:
+    branches:
+       - "**open-rc**"
+jobs:
+  build:
+    environment:
+      name: ${{ github.ref_name == 'open-release/palm.nelp' && 'prod' || 'stage' }}
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: "Echo job  vars"
+      env:
+        JOB_VARS: ${{ toJson(vars) }}
+      run: echo "$JOB_VARS"
+
+    - name: checkout mfe repo
+      uses: actions/checkout@v3
+
+    - name: Use Node.js ${{ vars.NODE_VERSION }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ vars.NODE_VERSION }}
+
+    - name: Cache node modules
+      id: cache-npm
+      uses: actions/cache@v3
+      env:
+        cache-name: cache-node-modules
+      with:
+        # npm cache files are stored in `~/.npm` on Linux/macOS
+        path: ~/.npm
+        key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-build-${{ env.cache-name }}-
+          ${{ runner.os }}-build-
+          ${{ runner.os }}-
+
+    - if: ${{ steps.cache-npm.outputs.cache-hit != 'true' }}
+      name: List the state of node modules
+      continue-on-error: true
+      run: npm list
+
+    - name: npm Install
+      run: npm install
+
+    - name: npm Build  # check env variables of repo.
+      run: npm run build
+      env:
+        PUBLIC_PATH: ${{ vars.PUBLIC_PATH_CDN }}
+        APP_ID: ${{ vars.APP_ID }}
+        MFE_CONFIG_API_URL: ${{ vars.MFE_CONFIG_API_URL }}
```

Comment:
This is going to point to stage or prod domains. But what could we do with alpha?
![image](https://github.com/user-attachments/assets/339e4369-6936-4587-bda9-570a23945afa)

Reply:
probably you are referring to this https://edunext.atlassian.net/browse/FUTUREX-1023  and yes, this will use stage as the other mfes